src/core/Akka/Util/Extensions/TaskExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TaskExtensions.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace System.Threading.Tasks
+{
+    internal static class TaskExtensions
+    {
+//#if NETSTANDARD
+        public static async Task WaitAsync1(this Task task, TimeSpan timeout)
```

Comment:
Need help with the `#if` targeting here or is this just commented out for debugging purposes?

Reply:
Reverted back